lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1402,3 +1425,53 @@ def _get_deployable_vhd_path(
             wait_copy_blob(blob_client, vhd_path, log)
 
         return full_vhd_path
+
+    def _node_generate_min_capability(
+        self, req_cap: schema.NodeSpace, azure_cap: schema.NodeSpace
+    ) -> schema.NodeSpace:
+        assert azure_cap.features, "azure node has no disk features."
+        if req_cap.features is None or (
+            not req_cap.features.__contains__(DiskType.DISK_EPHEMERAL)
```

Comment:
The standard Python grammar is `DiskType.DISK_EPHEMERAL not in req_cap.features`

Reply:
Thanks for catching! This was a leftover from when I was trying to use Disk names as enum. I will update this.

Reply:
I have updated few things here to make the code more readable. Please feel free to suggest if anything could be improved further.